packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -196,17 +187,14 @@ export default function ViewApplicationPage() {
   const [hasAccess, setHasAccess] = useState(true);
 
   useEffect(() => {
-    if (isApplicationFetched) {
-      setApplicationExists(!!application);
+    if (!isLoading) {
+      setApplicationExists(!getApplicationByIdError && !!application);
+
       if (round) {
-        round.operatorWallets?.includes(address?.toLowerCase())
-          ? setHasAccess(true)
-          : setHasAccess(false);
-      } else {
-        setHasAccess(true);
+        setHasAccess(!!round.operatorWallets?.includes(address?.toLowerCase()));
       }
     }
-  }, [address, application, isApplicationFetched, round]);
+  }, [address, application, isLoading, round]);
```

Comment:
might need to add `getApplicationByIdError` to dependencies list to follow eslint rules